packages/expo-updates/e2e/__tests__/setup/create-eas-project-basic.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+
+const { initAsync, setupBasicAppAsync } = require('./project');
+
+const repoRoot = process.env.EXPO_REPO_ROOT;
+const workingDir = path.resolve(repoRoot, '..');
+const runtimeVersion = '1.0.0';
+
+/**
+ * This generates a project at the location TEST_PROJECT_ROOT,
+ * that is configured to build a test app and run the "basic" suite
+ * of updates E2E tests in the Detox environment.
+ *
+ * To test this locally, export the following environment variables:
+ * $ export UPDATES_HOST=$(ifconfig -l | xargs -n1 ipconfig getifaddr)
+ * $ export UPDATES_PORT=4747
+ * $ export EXPO_REPO_ROOT=<path to local expo repo>
+ * $ export ARTIFACTS_DEST=<path to any temp artifacts dir>
+ * $ export TEST_PROJECT_ROOT=<path to any temp dir for the project root>
+ *
+ * Then execute this file to setup the test project and builds.
+ *
+ * Afterwards, tests can be run by changing to TEST_PROJECT_ROOT and running
```

Comment:
having a dumb question here: could we have a backward compatible change here? say if the tests on eas-build come with problems and i want to build-and-run the tests locally. how could i do this?

Reply:
This is certainly possible... I'll add instructions for how to do this

Reply:
Done